README.md

```diff
@@ -1,32 +1,298 @@
+   5. Click on "ADD DEPENDENCIES" again and add  "Lombok" from the list.
+   6. Click on "ADD DEPENDENCIES" again and add "Spring Data JPA" from the list.
+   7. Click on "ADD DEPENDENCIES" again and add "PostgreSQL Driver" from the
+      list.
+   8. Click on "ADD DEPENDENCIES" again and add "Validation" from the list.
+4. Click on the "Generate" button on the bottom. This will download a zip file
+   containing the Spring Boot Project.
+5. Unzip the archive and open it in IntelliJ or a preferred code editor.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+![spring-project-initializr-configurations](https://curriculum-content.s3.amazonaws.com/spring-mod-2/project/spring-initializr-project.png)
 
-## Topic 2
+In pgAdmin4, create the camp_db:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+![create-camp-db](https://curriculum-content.s3.amazonaws.com/spring-mod-2/project/pgadmin-create-camp-db.png)
 
-## Conclusion
+## Models
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+You need to create the following relationships:
 
-## Resources
+- A `Camper` can have many `Signups`.
+- An `Activity` can have many `Signups`.
+- A `Signup` can have a `Camper` for a certain `Activity`
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+Start by creating the models for the following database tables:
+
+![camper-er-diagram](https://curriculum-content.s3.amazonaws.com/spring-mod-2/project/camp-er-diagram.png)
+
+If you clone down this Git repository, you'll find the `data.sql` file that
+contains the schema you can import into pgAdmin4.
+
+## Project Requirements
+
+Create a Spring Boot API that models the campers signing up for activities.
+Make sure you consider the relationships, that are shown in the
+entity-relationship (ER) diagram above.
+
+It is recommended to follow this outline of how to go about creating the
+project:
+
+- Create all the classes and interfaces to set up the project structure.
+- Write the code for the entity and DTO classes first.
+    - For example: Create the `Activity`, `Camper` and `Signup` entities and then
+      write their corresponding DTO classes.
+    - Consider the relationships based on the ER diagram above.
+    - Consider the validations that should be added to the DTO classes in the
+      section below.
+- Add the `ModelMapper` bean to the configuration file
+  (i.e., `SpringProjectApplication`).
+  - Note: You will most likely need to add the `ModelMapper` dependency to the
+    `pom.xml`.
+- Add the database properties to the `application.properties` file to connect to
+  the PostgreSQL database, `camp_db`.
+- Write the code for the repository classes to extend the `CrudRepository`
```

Comment:
Should we have the students extend JpaRepository rather than CrudRepository?
(It provides a few dozen more built-in methods and shouldn't require any additional dependency)

Reply:
So far, in all the lessons, I've been using the `CrudRepository` interface. So everything they have to do in this project they _can_ accomplish with the `CrudRepository` interface. I can add maybe a sentence or provide more explanation about the `JpaRepository` interface in the original repositories lesson (back in Spring Mod 1) but that is if time allows at this point. By all means, if they extend the `JpaRepository` instead of the `CrudRepository`, I don't think they should be deducted any points on their project though.

Reply:
(But also, I can see how changing this to `JpaRepository` could simplify the `getCamper()` and other get methods way easier.